src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -22,6 +22,9 @@ public class JdbcDataDriver {
 	private CustomerDao custDao;
 	private MobileDao mobileDao;
```

Comment:
why you need mobileDao in Driver class?

Reply:
it is a variable for internally creating object

Reply:
> it is a variable for internally creating an object